### DISCUSSION TITLE
Consider adding additional type information - a Skip warning message

### DISCUSSION THREAD

**Discussion by davidakoontz** [skiptools/skip#185] [Q&A] [answered]
<img width="1009" alt="Screenshot 2024-08-13 at 5 32 52 PM" src="https://github.com/user-attachments/assets/b3bd30b0-55fb-422d-97e3-48dde9ded09d">
`struct iOSCheckboxStyle: ToggleStyle {
    func makeBody(configuration: Configuration) -> some View {

        Button(action: {

            configuration.isOn.toggle()
            
        }, label: {
            HStack {
                Image(systemName: configuration.isOn ? "checkmark.square" : "square")
                
                configuration.label
            }
        })
    }
}`

> Skip is unable to match this API call to determine whether it results in a View.  Consider adding additional type information.

OK - considering... but HOW?  

The configuration.label is a ToggleStyleConfiguration.Label  it is being returned in the View of the HStack.
Any idea how I could help the system out... how I'd add Type info the the return?

**Comment by marcprux** [comment, marked as answer]
The error message is vague, but we don't support `ToggleStyle` yet. You can have it on the iOS side by including it in a `#if !SKIP` block, but the Android side only supports the default toggle style currently.

You could probably fake it with a custom component with an `onTap` modifier.

**Reply by davidakoontz** [reply]
OK - thanks... I will insert another #if !SKIP  who needs properly styled toggle buttons on Android... they can figure it out.
